app/notifications/receive_notifications.py

```diff
@@ -30,6 +30,15 @@ def receive_mmg_sms():
     """
     post_data = request.get_json()
 
+    auth = request.authorization
+
+    if not auth:
+        current_app.logger.warning("Inbound sms (MMG) no auth header")
+        # abort(401)
+    elif auth.username != 'mmg_co_2017' or auth.password not in current_app.config['MMG_INBOUND_SMS_AUTH']:
```

Comment:
Should the username be an environment variable too? Its giving away half of the credentials.

Reply:
Will do in the PR that enables blocking.